packages/clay-time-picker/src/index.tsx

```diff
@@ -238,9 +238,33 @@ const ClayTimePicker: React.FunctionComponent<IProps> = ({
 				break;
 			default:
 				if (regex.test(event.key) && configName !== TimeType.ampm) {
+					const maxFirstDigit =
+						configName === TimeType.hours
+							? use12Hours
+								? 1
+								: 2
+							: 5;
```

Comment:
@bryceosterhaus just a doubt, I can't quite understand why this value is 5 here 😅.

Reply:
This is because we never want to allow the first digit in the minutes input to be greater than 5. It's the max value for the "10"s spot in minutes. That way we allow 0-59, but not 60+.

I guess we could do this programatically by taking the "10"s digit from the config.max